test/ably/restrequest_test.py

```diff
@@ -0,0 +1,77 @@
+import six
+
+from ably import AblyRest
+from ably.http.paginatedresult import HttpPaginatedResult
+from test.ably.restsetup import RestSetup
+from test.ably.utils import BaseTestCase
+from test.ably.utils import VaryByProtocolTestsMetaclass, dont_vary_protocol
+
+test_vars = RestSetup.get_test_vars()
+
+
+# RSC19
+@six.add_metaclass(VaryByProtocolTestsMetaclass)
+class TestRestRequest(BaseTestCase):
+
+    def setUp(self):
+        self.ably = AblyRest(key=test_vars["keys"][0]["key_str"],
+                             rest_host=test_vars["host"],
+                             port=test_vars["port"],
+                             tls_port=test_vars["tls_port"],
+                             tls=test_vars["tls"])
+
+        # Populate the channel (using the new api)
+        for i in range(50):
```

Comment:
I would consider reducing the total number down as this test may be brittle as it could hit the rate limits for these accounts. That, or add a very small delay in perhaps?